roles/common/tasks/main.yml

```diff
@@ -85,6 +86,18 @@
   file: name=/var/lib/rsyslog state=directory
   tags: rsyslog
 
+- name: Put rsyslog client cert
```

Comment:
Idea to make these conditional on rsyslog_remote_server being defined? This way existing (test/develop) environments don't have to add (dummy) keys and certs.